### PR TITLE
[WEBDEV-1025] Restructure data folder

### PR DESCRIPTION
**Pull request by naseberry**
- [WEBDEV-1039 [Augustus] Restructure of data view folder](https://ukparliament.atlassian.net/browse/WEBDEV-1039) Refactor data view folder
- [WEBDEV-1040 [Augustus] Agree and apply testing strategy](https://ukparliament.atlassian.net/browse/WEBDEV-1040) Testing strategy
- [WEBDEV-1041 [Augustus] - Create JSON for index page](https://ukparliament.atlassian.net/browse/WEBDEV-1041) Create json for root page
- [WEBDEV-1042 [Augustus] Move current homepage to new website folder](https://ukparliament.atlassian.net/browse/WEBDEV-1042) Move current homepage to website folder

### CODE REVIEW

File: data/_types/lists/index.json [hunk 1]
```diff
@@ -0,0 +1,167 @@
+{
+  "layout": {
+    "template": "layout"
+  },
+  "meta": {
+    "title": "Lists - Types - UK Parliament",
```
Comment (UsmanAfzal) [inline data/_types/lists/index.json]: Is this title correct?

Comment (naseberry) [reply data/_types/lists/index.json]: As these *.json files are for local development, I have decided to use meta.title in conjunction with our lighthouse audit. So in essence, yes meta.title are correct.

See ticket: WEBDEV-1060

File: data/_prototypes/index.json [hunk 2]
```diff
@@ -0,0 +1,167 @@
...
+                    "variant": "highlight"
+                  }
+                ]
+              },
+              "components": [
+                {
+                  "name": "paragraph",
+                  "data": [
+                    {
+                      "content": "There is currently no prototypes to show on this page."
```
Comment (UsmanAfzal) [inline data/_prototypes/index.json]: Change to something more generic to use everywhere. 'There are currently no views'

File: data/_types/lists/index.json [hunk 3]
```diff
@@ -0,0 +1,167 @@
...
+                    "variant": "highlight"
+                  }
+                ]
+              },
+              "components": [
+                {
+                  "name": "paragraph",
+                  "data": [
+                    {
+                      "content": "There is currently no prototypes to show on this page."
```
Comment (UsmanAfzal) [inline data/_types/lists/index.json]: Change to something more generic to use everywhere. 'There are currently no views'

File: data/_types/things/index.json [hunk 4]
```diff
@@ -0,0 +1,167 @@
...
+                    "variant": "highlight"
+                  }
+                ]
+              },
+              "components": [
+                {
+                  "name": "paragraph",
+                  "data": [
+                    {
+                      "content": "There is currently no prototypes to show on this page."
```
Comment (UsmanAfzal) [inline data/_types/things/index.json]: Change to something more generic to use everywhere. 'There are currently no views'

File: data/_types/utilities/index.json [hunk 5]
```diff
@@ -0,0 +1,167 @@
...
+                    "variant": "highlight"
+                  }
+                ]
+              },
+              "components": [
+                {
+                  "name": "paragraph",
+                  "data": [
+                    {
+                      "content": "There is currently no prototypes to show on this page."
```
Comment (UsmanAfzal) [inline data/_types/utilities/index.json]: Change to something more generic to use everywhere. 'There are currently no views'

File: test/fixtures/html/integration/_types/things/index.html [hunk 6]
```diff
@@ -0,0 +1,126 @@
...
+    <main>
+        <div class="section--primary" id="content">
+            <div class="container">
+                <h1><span>Things</span></h1>
+            </div>
+        </div>
+        <section class="section--major section--wide ">
+            <div class="container">
+                <div class="status--highlight ">
+                    <p>There is currently no prototypes to show on this page.</p>
```
Comment (UsmanAfzal) [inline test/fixtures/html/integration/_types/things/index.html]: Change to something more generic to use everywhere. 'There are currently no views'

File: test/fixtures/html/integration/_types/lists/index.html [hunk 7]
```diff
@@ -0,0 +1,126 @@
...
+    <main>
+        <div class="section--primary" id="content">
+            <div class="container">
+                <h1><span>Lists</span></h1>
+            </div>
+        </div>
+        <section class="section--major section--wide ">
+            <div class="container">
+                <div class="status--highlight ">
+                    <p>There is currently no prototypes to show on this page.</p>
```
Comment (UsmanAfzal) [inline test/fixtures/html/integration/_types/lists/index.html]: Change to something more generic to use everywhere. 'There are currently no views'

File: data/_types/lists/index.json (hunk 1 above)
Comment (naseberry) [reply data/_types/lists/index.json]: Got you, we wanted to reflect the path of the file `./_types/lists`, however we can change this if required. Note it is also the same for `./_types/things` and `./_types/utilities`

**Pull request merged**